Labs/Alyssa_Giedd_Lab2.ipynb

```diff
@@ -0,0 +1,477 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x2271168b780>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "While at first glance the results appear like some variation of gaussian, we can more clearly see on the semilog plot that the distribution is very much NOT a gaussian one, and would require beyond 100 intervals to approach one.\n"
+     ]
+    }
+   ],
+   "source": [
+    "#B) Discuss how the shape changes. Does it approach a Gaussian distribution? If yes, after how many intervals?\n",
+    "days = [1, 2, 5, 8, 15, 100]\n",
+    "\n",
+    "for index in range(len(days)):\n",
+    "    #Sums to plot\n",
+    "    bg_summation = convolve_multiple(y_avg, days[index])\n",
+    "    #Produce graph with the specified days worth of data\n",
+    "    plt.semilogy(np.linspace(0, len(bg_summation), len(bg_summation))/days[index], bg_summation, label = str(days[index]) + \" day sum\")\n",
+    "\n",
+    "plt.legend()\n",
+    "plt.title(\"Average Background for a Range of Days\")\n",
+    "plt.show()\n",
+    "\n",
+    "#explanation\n",
+    "print(\"While at first glance the results appear like some variation of gaussian, we can more clearly see on the semilog plot that the distribution is very much NOT a gaussian one, and would require beyond 100 intervals to approach one.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem 3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 124,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The probability of getting a signal of 12.6 or higher is:  0.05762822227615316 With sigma:  1.575\n",
+      "Therefore, a discovery cannot be claimed.\n"
+     ]
+    }
+   ],
+   "source": [
+    "#A) From looking at all the other pixels in your image, you can measure the width of the background Gaussian distribution X \n",
+    "#(pick something). Assuming you see a signal of strength Y (pick a floating-point number; optical CCDs are not sensitive \n",
+    "#enough to count photons so the readings are floating point brightnesses, not integer photons). \n",
+    "#Calculate the significance of your detection. Can you claim a discovery (traditionally 5-sigma or more)?\n",
+    "width = 8\n",
+    "signal = 12.6\n",
+    "x = np.linspace(-13, 13, 1000)\n",
+    "bg = stats.norm.pdf(x,loc = 0, scale =8)\n",
+    "\n",
+    "prob = stats.norm.cdf(12.6, loc = 0, scale = width)\n",
+    "sigma = stats.norm.ppf(prob)\n",
+    "\n",
+    "print(\"The probability of getting a signal of 12.6 or higher is: \" , 1 - prob,\"With sigma: \" , sigma)\n",
```

Comment:
before this calculation consider including the statistical question and integral to indicate what you are calculating. this is purely a suggestion on my end because the lab instructions don't say to do it (so up to your discretion!)